### DISCUSSION TITLE
The artifact is created and uploaded successfully, but failed to download in next step.

### DISCUSSION THREAD

**Discussion by EltonChou** [community/community#81545] [Actions] [answered]
### Select Topic Area

Question

### Body

I didn't modify the artifact related workflow since last month and I have tried serveral times, but it not works.

The [REPO](https://github.com/EltonChou/TwitterMediaHarvest) is public.

Last month: https://github.com/EltonChou/TwitterMediaHarvest/actions/runs/6994383133
![image](https://github.com/community/community/assets/12560310/890aba38-9bdf-4eeb-b6be-5702d438a2b3)
![image](https://github.com/community/community/assets/12560310/701f420a-2ed5-4e2a-8fa5-4c0c206aa523)

This month: https://github.com/EltonChou/TwitterMediaHarvest/actions/runs/7306496038
![image](https://github.com/community/community/assets/12560310/7b2e7dbd-5c77-44f7-abb5-c948c8f3bb1e)
![image](https://github.com/community/community/assets/12560310/f15b5f39-59f3-488b-af83-90410140b4fc)

**Comment by ChristopherHX** [comment, marked as answer]
You have mixed of upload-artifact@v3 and download-artifact@v4 also known as download-artifact@master at this time.

https://github.com/actions/download-artifact/releases/tag/v4.0.0

v3 and v4 (aka master) are not compatible, consider using v4 everywhere. Otherwise use v3 everywhere.

**Reply by EltonChou** [reply]
Thanks a lot!
I didn't notice that, I fixed the version and it is working.
